tools/jenkins/docker_run.sh

```diff
@@ -42,6 +42,8 @@ else
   cp -r "$EXTERNAL_GIT_ROOT/$RELATIVE_COPY_PATH"/* "/var/local/git/grpc/$RELATIVE_COPY_PATH"
 fi
 
+[ -e /post-git-setup.sh ] && /post-git-setup.sh
```

Comment:
docker_run.sh is not used for run_tests.py test runs (it is used for task_runner.py), so this line is a bit misleading.
(I was planning to cleanup and unify the shell scripts eventually, but didn't find enough time to do that yet).